BranchGuide.md

```diff
@@ -0,0 +1,43 @@
+# Mixed Reality Toolkit Branch guide
+
+The Mixed Reality Toolkit uses multiple working branches within the project to manage the various components of delivery, these can be broken down to:
+
+---
+## Release Branches
+
+Release branches are frozen code linked to a specific released asset, the code held within these branches should match 1-1 with their corresponding asset.
+This ensures whether you download the asset or the code, there are no differences in the delivery.
+
+* **[master](https://github.com/Microsoft/MixedRealityToolkit-Unity) - [HoloToolkit 2017.4.2.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/2017.4.2.0)** - Current HoloToolkit release branch.
+
+* **[mrtk_release](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release) - [Mixed Reality Toolkit 2018.9.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/2018.9.0-Beta) (vNext Beta)** - Current Mixed Reality Toolkit (vNext) beta release.
+
+---
+## Development Branches
+
+Development branches relate to the current active and tested development state of the product.  These should be considered "stable", however bugs may be introduced during the course of development.
+
+* **Dev_Working_Branch/[htk_development](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/htk_development)** - Legacy HoloToolkit development - Currently frozen and due to be deprecated.
+
+* **[mrtk_development](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_development)** - Current Mixed Reality Toolkit (vNext) active development
+
+---
+## Feature Branches
+
+These relate to new components / features that are actively being developed for consideration to be included in the main development branch.
+These should not be considered stable and main contain breaking features, **use at your own risk**.
+
+> This list may not be up to date as development during the vNext beta is quite rapid, so simply use this as a guide and [ask in Slack](https://holodevelopersslack.azurewebsites.net/) regarding future features / capabilities.
+
+* [feature/LuminPlatform](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/feature/LuminPlatform) - (@StephenHodgson) Experimental Magic Leap device integration.
```

Comment:
This list will quickly go out of date. It may be advisable to show an example of how feature branches work here.

Otherwise, this file will need to be modified whenever a branch is added or deleted -- an easy task to forget.

Reply:
So what is your suggestion exactly?  I simply listed the current branches as examples, doesn't need to be an exhaustive list and we can update this list from time to time with large pieces of work.

I've added a statement to say these are examples and reduced the number of features listed to just the main ones.

Reply:
Perhaps have one that does not actually exist, but serves as an example of the pattern we use for feature branches. 

Not sure if it needs to list owners. Those may change or be more than one person.